src/services/permission/permission.service.ts

```diff
@@ -39,13 +39,15 @@ export class PermissionService {
   private async checkWritePermission(auth: RepoAuth, gitUrl: string): Promise<boolean> {
     const response = await this.http.fetch(`${gitUrl}/${GitServices.Push}`, {
       headers: this.getHeaders(auth),
+      method: "POST",
```

Comment:
Why do we need to specify it explicitly? What's the default method?

Reply:
Hi @raych1, the default method is GET and Currently we cannot get work item after user create work branch. It is due to comparing commits api from git-rest-api failed. After our debug, we found the checking write permission api not support currrent method. So we want to change the method to POST. Now, we also have the issue on TEST and PROD env.

Reply:
Can you help me understand why this is not working but worked before? What change caused this issue?